supervision/detection/utils/iou_and_nms.py

```diff
@@ -231,6 +231,96 @@ def box_area(box):
     return ious
 
 
+def box_iou_batch_alt(
```

Comment:
Let's rename it to `box_iou_batch` and remove old implementation of this function.